test/e2e/csc_tests.go

```diff
@@ -18,21 +21,22 @@ const (
 
 	// targetNamespace is the non-existing target namespace
 	targetNamespace string = "non-existing-namespace"
+
+	// nonExistingPackageCSCName is the name of the catalogsourceconfig that points
+	// to a non-existing Package
+	nonExistingPackageCSCName string = "non-existing-package-operators"
+
+	// packageName is the non-existing package
+	packageName string = "non-existing-package"
```

Comment:
These shouldn't be global constants as they are only being used in one test case. I now realize `targetNamespace` might be setting a bad pattern.